### PR TITLE
Fix the README file

### PR DESCRIPTION
**Pull request by nfroidure**
Find out that i missed to apply some changes to the readme file.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -19,9 +19,11 @@ const firstChunkStream = require('first-chunk-stream');
 
 // unicorn.txt => unicorn rainbow
 fs.createReadStream('unicorn.txt')
-	.pipe(firstChunkStream({chunkLength: 7}, function (chunk, enc, cb) {
-		this.push(chunk.toUpperCase());
-		cb();
+	.pipe(firstChunkStream({chunkLength: 7}, function (err, chunk, enc, cb) {
```
Comment (sindresorhus) [inline readme.md:22]: @nfroidure Just curious. Why does the user have to handle the error? I know it's convention for callbacks to be error-first, but this is not a normal callback interface.

Comment (nfroidure) [inline readme.md:22]: Assumed the user would like to know when there is an error in the process of retrieving the first chunk without having to add extra code to recognize those errors.

Comment (sindresorhus) [inline readme.md:22]: What kind of error? We already require the user to check the data length to ensure it's big enough.

Comment (nfroidure) [inline readme.md:22]: Any io error that a stream can throw, mainly broken pipes, out of memory errors etc..

Comment (nfroidure) [inline readme.md:22]: The various use cases i thought of are there:
Silently fail:
https://github.com/sindresorhus/first-chunk-stream/blob/master/test.js#L43
Casting the error:
https://github.com/sindresorhus/first-chunk-stream/blob/master/test.js#L72
Normal behavior when errors happen after the first chunk has been found:
https://github.com/sindresorhus/first-chunk-stream/blob/master/test.js#L107

Comment (sindresorhus) [inline readme.md:22]: Oh right. Forgot about those. All good.

**Pull request closed without merging**